### DISCUSSION TITLE
VideoWrite file size ballooned

### DISCUSSION THREAD

**Discussion by nghess** [bonsai-rx/bonsai#1630] [Q&A]
Hi,
I came across a strange issue where 40 minute .avi files saved by VideoWriter ballooned from ~3gb to ~12gb suddenly. To the best of my knowledge, nothing in the Bonsai file was changed, but after 2 months of recording files around 3gb the file size blew up. As far as I can tell, the video quality and framerate is the same across all the files regardless of size. Can anyone offer insight?

Thanks for the help!

**Comment by bruno-f-cruz** [comment]
Share the workflow as suggested please!

**Reply by nghess** [reply]
Sorry! Here it is:
https://github.com/nghess/fmon-bpod/blob/main/bonsai%20protocols/bpod-fmon.bonsai

**Reply by bruno-f-cruz** [reply]
Assuming you didn not chat anything in the workflow, is it possible that your camera is now recording color instead of mono data?

**Reply by nghess** [reply]
Visual inspection tells me the files are identical. And in file details, both framerate and color depth haven't changed. I seems I just lost .avi compression at some point. I am guessing that this has something to do with Windows updates or updates to camera software, considering that the Bonsai workflow didn't change. --But on the off-chance that it was a known issue in Bonsai I figured I'd ask.

**Reply by bruno-f-cruz** [reply]
Without knowing what you changed it is going to be tough. Are you maybe using a different codec to encode the video? Or has the camera (not the encoding) frame rate changed?
